R/lineage.R

```diff
@@ -15,55 +15,21 @@
 #'
 #' @export
 
-LineageTree <- function(object, clonotype, regions = "V", reference, chain = c("VDJ", "VJ"), color.tip.by = NULL, clonotype.column = NULL) {
+LineageTree <- function(object, clonotype, regions = "V", reference, chain = c("VDJ", "VJ"), color.tip.by = NULL, clonotype.column = "clonotype") {
     if (DefaultAssayVDJ(object) != "BCR") {
-        stop("Lineages can only be computed on BCR data!", call. = F)
+        print("Lineages can only be computed on BCR data!", call. = F)
```

Comment:
Why don't we stop here anymore? This function should only be used on BCR data, since the biological relevance for TCR data is very limited/none.